sdk/identity/Azure.Identity/api/Azure.Identity.net8.0.cs

```diff
@@ -186,9 +186,11 @@ public CredentialUnavailableException(string message, System.Exception innerExce
     }
     public partial class DefaultAzureCredential : Azure.Core.TokenCredential
     {
+        public static readonly string DefaultEnvironmentVariableName;
```

Comment:
I now wonder if this should not be a const. Const would allow us to use this value as a default for optional parameters. I know we cannot do it in the ctor of this class. But I was just writing some code and wanted a const like this.